src/GraphBuilder.js

```diff
@@ -22,10 +22,13 @@ export default class GraphBuilder {
     // A relative URI to use as the base
     const baseURI = rdf.namedNode('')
 
-    // Is there ever more than one base node?
-    Object.keys(this.state).forEach((key) => {
-      this.buildTriplesForNode(baseURI, rdf.namedNode(this.state[key].rdfClass), this.getPredicateList(key))
+    Object.keys(this.state).forEach((rtId) => {
+      // Assuming that loadTimestamp will be close enough for included template
+      // and so only will record for main template.
```

Comment:
Not blocking but what would it take to record this timestamp for all of the embedded resources in the  main template's propertyTemplates?

Reply:
Yes, but going to hold pending #619.